### DISCUSSION TITLE
Can Symphonia be used for simply reading metadata?

### DISCUSSION THREAD

**Discussion by kuraiji** [pdeljanov/Symphonia#132] [General]
I know that the main page mentions "Read most metadata and tagging formats", but would this library be overkill for simply reading metadata?

**Comment by pdeljanov** [comment]
Hi @kuraiji,

I think Symphonia would be a bit overkill if you just need to read the metadata. Check out [`lofty-rs`](https://github.com/Serial-ATA/lofty-rs) instead which looks like a perfect fit for your use-case.

Symphonia may be useful if you want additional information such as track listings and duration (if not written to the tag).
